src/main/java/net/minecraftforge/fml/common/registry/VillagerRegistry.java

```diff
@@ -276,15 +301,23 @@ private void register(VillagerCareer career)
         private VillagerProfession profession;
         private String name;
         private int id;
+        private EntityVillager.ITradeList[][] trades; // (dvatreknerd314) stores trades with career objects
         public VillagerCareer(VillagerProfession parent, String name)
         {
             this.profession = parent;
             this.name = name;
             parent.register(this);
         }
+        
+        // Returns static copy of trades available to this career - dvatreknerd314
+        public EntityVillager.ITradeList[][] getTrades()
+        {
+            return static trades;
+        }
```

Comment:
Again, giving direct access to this will derp things.
What you'd want to do is keep these trades in a private list and have the vanilla code access it in a READONLY manor.